LLama/Native/SafeLLamaContextHandle.cs

```diff
@@ -809,7 +809,8 @@ public int KvCacheCountTokens()
         /// </summary>
         public void KvCacheClear()
         {
-            NativeApi.llama_kv_self_clear(this);
+            //NativeApi.llama_kv_self_clear(this);
```

Comment:
Not sure why this change was made? `llama_kv_cache_clear` was renamed `llama_kv_self_clear` in llama.cpp recently, so that should always be used instead now.

Reply:
Yes, for some reason there are/were problems with the llama.cpp submodule. I had a slightly lower version (after downloading a version initially) and that version of llama.cpp does not have the llama_kv_self_clear yet. This was the reason for why I have reintroduced the old clear. I think that it would be better to keep it still in the code for this reason (it is marked as obsolete), but it is up to you to decide.

Reply:
That was probably me forgetting to update the submodule again on the last update. If you keep an eye on the update PRs when they're available feel free to give me a poke if you spot that in the future.

For `llama_kv_cache_clear` vs `llama_kv_self_clear`, please put it back to the newer version. If it was a `public` API it'd make sense to keep it around for a while marked as `Obsolete` to give people a chance to migrate, but this is an `internal` function.